app/client/components/checkFlowIsValid.tsx

```diff
@@ -9,13 +10,20 @@ const renderChildrenIfValidated = (props: CheckFlowIsValidProps) => (
   props.validator(me) ? (
     <>{props.children}</>
   ) : (
-    <PageContainer>{props.noProductRenderer}</PageContainer>
+    <PageContainer>
+      <NoProduct
+        inTab={false}
+        supportRefererSuffix={props.supportRefererSuffix}
+        productType={props}
+      />
+    </PageContainer>
   );
 
 export type MeValidator = (me: MeResponse) => boolean;
 
 export interface CheckFlowIsValidProps extends ProductType {
   children: any;
+  supportRefererSuffix: string;
```

Comment:
Does this get used for tracking or something? We could use a type here?

Reply:
this is appended on the end of other strings which are used in tracking, so collating the possibilities into a type won't really add anything here

Reply:
Thanks for explaining this 👍